README.md

```diff
@@ -133,7 +133,13 @@ The OPA-Istio plugin supports the following configuration fields:
 | Field | Required | Description |
 | --- | --- | --- |
 | `plugins["envoy_ext_authz_grpc"].addr` | No | Set listening address of Envoy External Authorization gRPC server. This must match the value configured in the Envoy Filter resource. Default: `:9191`. |
-| `plugins["envoy_ext_authz_grpc"].query` | No | Specifies the name of the policy decision to query. The policy decision must be return a `boolean` value. `true` indicates the request should be allowed and `false` indicates the request should be denied. Default: `data.istio.authz.allow`. |
+| `plugins["envoy_ext_authz_grpc"].query` | No | Specifies the name of the policy decision to query. The policy decision must return a `boolean` value. `true` indicates the request should be allowed and `false` indicates the request should be denied. Default: `data.istio.authz.allow`. **Note: This field will be deprecated soon. Use `boolean_query` instead.**|
+| `plugins["envoy_ext_authz_grpc"].boolean_query` | No | Specifies the name of the policy decision to query. The policy decision must return a `boolean` value. `true` indicates the request should be allowed and `false` indicates the request should be denied. Default: `data.istio.authz.allow`.|
+| `plugins["envoy_ext_authz_grpc"].response_query` | No | Specifies the name of the policy decision to query. The policy decision must be an object containing the `status` key set to either `true` or `false` to indicate if the request is allowed or not respectively. The policy decision can optionally contain a `headers` field to send custom headers to the downstream client or upstream. Also an optional `body` field can be included in the policy decision to send a response body data to the downstream client.|
+
+The `boolean_query` (or `query`) and `response_query` fields are mutually exclusive. We recommend using the `boolean_query` field instead of the soon to be deprecated `query` field.
```

Comment:
nit: Should we maybe say something like "We recommend using the `boolean_query` or `response_query` field instead of the soon to be deprecated `query` field"? If I understand correctly we just want people to switch to the newer config options, right?

Reply:
`boolean_query` and `response_query` are the new fields. `boolean_query` is the replacement for the `query` option.  Will re-word this sentence.